packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -492,6 +492,35 @@ export type InsertableField<TSchema extends ImplicitFieldSchema | UnsafeUnknownS
 		? InsertableContent | undefined
 		: never;
 
+/**
+ * Content which could be read from a field within a tree.
+ *
+ * @remarks
+ * Extended version of {@link TreeFieldFromImplicitField} that also allows {@link (UnsafeUnknownSchema:type)}.
+ * Since reading from fields with non-exact schema is still safe, this is only useful (compared to {@link TreeFieldFromImplicitField}) when the schema is also used as input and thus allows {@link (UnsafeUnknownSchema:type)}
+ * for use
+ * @system @alpha
+ */
+export type ReadableField<TSchema extends ImplicitFieldSchema | UnsafeUnknownSchema> =
+	TSchema extends ImplicitFieldSchema
+		? TreeFieldFromImplicitField<TSchema>
+		: TreeLeafValue | TreeNode;
+
+/**
+ * Content which could be read from a field within a tree.
+ *
+ * @remarks
+ * Extended version of {@link TreeFieldFromImplicitField} that also allows {@link (UnsafeUnknownSchema:type)}.
+ * Since reading from fields with non-exact schema is still safe, this is only useful (compared to {@link TreeFieldFromImplicitField}) when the schema is also used as input and thus allows {@link (UnsafeUnknownSchema:type)}
+ * for use
+ * @system @alpha
+ */
+export type ReadSchema<TSchema extends ImplicitFieldSchema | UnsafeUnknownSchema> = [
```

Comment:
What do you think about `ReadableSchema`? It matches the other one, and "ReadSchema" sounds to me like a verb before it sounds like an adjective.

Reply:
This is not schema which can be read (like ReadableField is a filed type modified for reading). It is a schema you can use when reading.

Thus I think the meaning of the prefix is different in these two cases, so making them align seems harmful. That said I'm not a bug fan of this name but both are alpha and system, so we will API review them, and we can rename them even after they go public (unless we make them non system)

Reply:
I see. Okay, good enough for now then.